src/ImageSharp/Processing/Processors/Overlays/GlowProcessor.cs

```diff
@@ -65,25 +71,36 @@ protected override void OnApply(ImageBase<TPixel> source, Rectangle sourceRectan
                 startY = 0;
             }
 
+            int width = maxX - minX;
+            using (Buffer<TPixel> rowColors = new Buffer<TPixel>(width))
             using (PixelAccessor<TPixel> sourcePixels = source.Lock())
             {
+                for (int i = 0; i < width; i++)
+                {
+                    rowColors[i] = glowColor;
+                }
+
                 Parallel.For(
```

Comment:
`Vector4BlendTransforms` is now gone and everything uses the `PixelBlenders` and thus `GraphicOptions` to set them.